docs/experimental-features/user-risk-score.asciidoc

```diff
@@ -0,0 +1,74 @@
+[[user-risk-score]]
+== User risk score
+
+NOTE: This feature is available for {stack} versions 8.3.0 and newer.
+
+The user risk score feature highlights risky usernames from within your environment. It utilizes a transform with a scripted metric aggregation to calculate user risk scores based on alerts  generated within the past 90 days. The transform runs hourly to update the score as new alerts are generated.
+
+Each alert's contribution to the user risk score is based on the alert's risk score (`signal.rule.risk_score`). The risk score is calculated using a weighted sum where rules with higher time-corrected risk scores also have higher weights. Each risk score is normalized on a scale of 0 to 100.
+
+The following table shows how risk levels are applied to a username, based on the normalized risk score:
+
+[width="100%",options="header"]
+|==============================================
+|Risk level |User risk score
+
+|Unknown |< 20
+|Low |20-40
+|Moderate |40-70
+|High     | 70-90
+|Critical  | > 90
+
+|==============================================
+
+[discrete]
+[[deploy-user-risk-score]]
+=== Deploy the user risk score package
+
+To deploy the user risk score framework in your environment, follow https://github.com/elastic/detection-rules/blob/main/docs/experimental-machine-learning/user-risk-score.md[these steps]. These instructions also include steps to enable the `riskyUsersEnabled` feature flag.
+
+NOTE: Update user risk score artifacts after you upgrade the {stack}. To do this, download a release bundle that's compatible with the new {stack} version and repeat all the steps referenced above. Failure to do so might cause views in the {security-app} to break.
+
+NOTE: To view user risk score data in the {security-app}, you must https://github.com/elastic/detection-rules/blob/main/docs/experimental-machine-learning/user-risk-score.md#8-enable-kibana-features[enable the `riskyUsersEnabled` feature flag]. However, enabling the feature flag is *not* required to view the Lens dashboards.
+
+[[view-user-risk-score]]
+[discrete]
+=== View user risk score data
+
+If the `riskyUsersEnabled` feature flag is enabled:
+
+. In the {security-app}, go to *Explore* -> *Users*.
+. On the *Users* page, select the *User risk* tab.
+
+The User risk table shows a list of user names, their assigned risk score, and user risk classification. The table appears in reverse chronological order, with the most recent authenticated user at the top. Select the *User risk classification* menu to filter the table by the classification type.
+
+[role="screenshot"]
+image::images/urs-table.png[User risk score table]
+
+NOTE: The User risk table is *not* affected by the KQL time range.
```

Comment:
This note seems a natural continuation of the paragraph above the image, so I'd have them swap places -- otherwise the image breaks up the flow of info:
```suggestion
NOTE: The User risk table is *not* affected by the KQL time range.

[role="screenshot"]
image::images/urs-table.png[User risk score table]
```